src/molecules/formfields/ToggleField/index.js

```diff
@@ -22,17 +22,17 @@ const renderChoices = (choices, input) => {
   if (!choices) return null;
 
   const renderChoice = (choice, idx) => {
-    const variantName = () => toBoolean(input.value) === toBoolean(choice.value) ? 'toggle-selected' : 'toggle';  // eslint-disable-line
+    const variantName = toBoolean(input.value) === toBoolean(choice.value) ? 'toggle-selected' : 'toggle';
```

Comment:
Just a minor cleanup here, since this didn't need to be a function (and we can remove the `eslint-ignore`)